frontend/src/main.tsx

```
@@ -4,11 +4,23 @@ import "./index.css";
 import App from "./App.tsx";
 import { Provider } from "react-redux";
 import { store } from "./app/store.ts";
+import { BrowserRouter, Route, Routes } from "react-router";
+import { TableView } from "./components/table_view.tsx";
+import { Layout } from "./layouts/layout.tsx";
+import { CanvasView } from "./components/graph/canvas_view.tsx";
 
 createRoot(document.getElementById("root")!).render(
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)